dashboard/app/models/pd/workshop.rb

```diff
@@ -74,6 +74,19 @@ class Pd::Workshop < ActiveRecord::Base
     ]
   }
 
+  # Section types by course
+  SECTION_TYPE_MAP = {
+    COURSE_CSF => 'csf_workshop',
+    COURSE_CSP => 'csp_workshop',
+    COURSE_ECS => 'ecs_workshop',
+    COURSE_CS_IN_A => 'csina_workshop',
+    COURSE_CS_IN_S => 'csins_workshop',
+    COURSE_CSD => 'csd_workshop',
+    COURSE_COUNSELOR => 'counselor_workshop',
+    COURSE_ADMIN => 'admin_workshop'
+  }.freeze
+  SECTION_TYPES = SECTION_TYPE_MAP.values.freeze
```

Comment:
Rereading... Does it make sense to call this `WORKSHOP_TYPE_MAP` and `WORKSHOP_TYPE`?